src/gui/settings.cpp

```diff
@@ -85,7 +85,6 @@ const char* snCores[]={
 };
 
 const char* saaCores[]={
-  "MAME",
```

Comment:
Array size is still 2 as used in the option...

Reply:
to be honest, this option can go, unless you plan alternative SAA emulation core?

Reply:
No plans so far. I considered adding the Eulous SAA1099 core, but since SAASound is pretty accurate I have dropped the idea.